packages/node_modules/@ciscospark/widget-message/src/container.js

```diff
@@ -330,11 +338,12 @@ export class MessageWidget extends Component {
    * Attempts to fetch and establish a conversation
    *
    * @param {object} props
+   * @param {object} prevProps
    *
    * @memberOf MessageWidget
    *
    */
-  establishConversation(props) {
+  establishConversation(props, prevProps) {
```

Comment:
I don't understand the concern here. Instead of adding autobinding to this function, I passed the props in from the call of the function. IMO `props` vs `this.props` in the function would be confusing for me to read.